src/wallet/wallet.h

```diff
@@ -496,13 +496,14 @@ class CWalletTx
         ABANDONED
     };
 
-    /* Confirmation includes tx status and a pair of {block hash/tx index in block} at which tx has been confirmed.
-     * This pair is both 0 if tx hasn't confirmed yet. Meaning of these fields changes with CONFLICTED state
-     * where they instead point to block hash and index of the deepest conflicting tx.
+    /* Confirmation includes tx status and a triplet of {block hash/block height/tx index in block} at which tx has been confirmed.
+     * This triplet is whole 0 if tx hasn't confirmed yet. Meaning of these fields changes with CONFLICTED state
```

Comment:
In commit "Add block_height field in struct Confirmation" (de8fe7d717042823f2aa429be2a260904a10b42e)

Maybe replace "hasn't confirmed yet" with "is unconfirmed or abandoned" to be more specific and include mention of the abandoned state.